### ISSUE TITLE
Simplify changing "probe_position" and "probe_state" (between "positioned" and "parked") from scripts

### ISSUE THREAD

**Issue by Brow71189** [nion-software/nionswift-instrumentation-kit#12]

**Comment by Brow71189** [comment]
Yes, I can confirm that I can switch the positioned state off by setting `probe_position` to `None`.

`probe_state` always shows `parked` for me though, which is a  bit confusing.

**Comment by cmeyer** [comment]
This works as expected - probe state is either 'parked' or 'scanning'. If 'parked' it can be in the default position (None) or in the specified position (x, y). If you think we should have it another way, either re-open this or file a new issue. Thanks!
